app/serializers/project_serializer.rb

```diff
@@ -0,0 +1,7 @@
+class ProjectSerializer < ActiveModel::Serializer
```

Comment:
Actually looking at the docs it looks like if the controllers are namespaced it will look for the serializers in the same namespace.
From the docs:
```
When serializing a model inside a namespace, such as Api::V1::Post, ActiveModelSerializers will expect the corresponding serializer to be inside the same namespace (namely Api::V1::PostSerializer).
```